pubspec.yaml

```diff
@@ -7,21 +7,21 @@ funding:
 
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: '>=3.0.0 <4.0.0'
```

Comment:
```suggestion
  sdk: ^3.0.0
```

Reply:
Should also have changelog entries for this update – and the support for latest pkg:http

Reply:
let me know how the change log looks!